rpm_spec/qmgr.spec

```diff
@@ -11,11 +9,19 @@ Group:		Qubes
 Vendor:		Invisible Things Lab
 License:	GPL
 URL:		http://fixme
-Requires:	python, PyQt4, qubes-core-dom0-linux >= 2.0.22, qubes-core-dom0 >= 3.0.18
-Requires:	pmount, cryptsetup, wmctrl
-Requires:	dbus
+Requires:	python3
+Requires:	python3-PyQt4
+Requires:	python3-inotify
+Requires:	qubes-core-dom0-linux >= 2.0.22
+Requires:	qubes-core-dom0 >= 3.0.18
 Requires:	qubes-artwork
-BuildRequires:	PyQt4-devel
+Requires:	pmount
+Requires:	cryptsetup
```

Comment:
Could you drop them yourself as part of redoing backups? Backup code is still there, so I think it should go kind of together.